packages/lib-classifier/src/components/Classifier/components/Layout/components/MaxWidth/MaxWidth.js

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components'
+import { pxToRem, withResponsiveContext } from '@zooniverse/react-components'
+import { Box, Grid } from 'grommet'
+
+import { withStores } from '@helpers'
+import Banners from '@components/Classifier/components/Banners'
+import FeedbackModal from '@components/Classifier/components/Feedback'
+import ImageToolbar from '@components/Classifier/components/ImageToolbar'
+import MetaTools from '@components/Classifier/components/MetaTools'
+import QuickTalk from '@components/Classifier/components/QuickTalk'
+import SubjectViewer from '@components/Classifier/components/SubjectViewer'
+import TaskArea from '@components/Classifier/components/TaskArea'
+
+const ContainerGrid = styled(Grid)`
+  position: relative;
+
+  // fluid sizing on tablets
+  @media screen and (min-width: 769px) and (max-width: 1160px) {
```

Comment:
```suggestion
  @media screen and (min-width: 769px) and (max-width: 70.666rem) {
```

The upper limit of this media query depends on the browser’s base font size. It’s 1160px at 16px but will be proportionally larger at larger font sizes. 

Needs testing in large text mode to make sure the column widths respond appropriately.

Reply:
The lower limit is a physical display size, regardless of the text size that we’ve set in the browser. 

In mobile Safari, you can test this with the text size controls in reader mode.

Reply:
iPhone with 200% text (32px base font size.) The banner text is cramped. 

![A PH-TESS light curve with large text. The subject banner text is quite cramped.](https://user-images.githubusercontent.com/59547/215278421-5bb98284-c974-4597-b654-0ce4b3837ed9.jpeg)

Reply:
At larger text sizes, the banner should maybe stack vertically rather than horizontally.